spec.bs

```diff
@@ -1555,13 +1556,14 @@ This specification does not (yet) include mechanisms for callers to
 associate language or direction metadata with the displayable strings
 they provide as input to the API (e.g., {{PaymentCredentialInstrument/displayName}}).
 
-In the meantime, callers of the API should consider the following:
+In the meantime, callers of the API should:
 
 * Aim for consistency between values of {{SecurePaymentConfirmationRequest/locale}}
     (when provided) and the language of displayable strings.
 
-* If displayable strings contain or might contain [=bidirectional text=], ensure
-    that they will display correctly without the application needing to process
-    the contents of the string value via the insertion of Unicode bidirectional
-    controls. For more information see [How to use Unicode controls for bidi
-    text](https://www.w3.org/International/questions/qa-bidi-unicode-controls).
+* Ensure that displayable strings that contain (or might contain)
+    [=bidirectional text=] will display correctly without the Relying
+    Party (or other [=consumer=] of these strings) needing to apply additional
```

Comment:
Hi Stephen,
I think of the RP as the party that validates the assertion (and the strings therein). In some cases the RP may also be the producer of the strings (e.g., 3DS), but I don't know that that will always be the case. 

It might be that other parties than the RP also need the strings; I don't know in the general case.

That, at least, is what I had in mind. 

I was not thinking about the UA as the consumer of the strings. If we consider that they are also consumers of the strings, then we could also mention them.

Reply:
Incidentally, the fact that the Relying Party has to validate the assertion with the strings is one concern I have with this idea to insert bidirectional controls (will they show up in some form in the cryptogram? will that be confusing? I have no idea 🤷 ).

On the main question of who is the consumer - I had assumed that for bidirectional needs only anyone who is displaying the strings cares about bidirectionality, which would imply the UA to me...

Reply:
No need to apologize for close review! @stephenmcgruer and @aphillips, I have tweaked the text again; many thanks for your reviews.